### PR TITLE
docs: add PR template

### LINKED ISSUE #76: Setup Pull-Request template
Create a Pull-Request template for new PRs that includes the following:
- description
- checklist for common thinks that need to be checked (documentation etc.)

### PR DESCRIPTION
**Pull request by larsrickert**
closes #76

## Description

Add a Pull request template (the same that is used for this pull request)

## Checklist

- [x] The added / edited code has been documented with [JSDoc](https://jsdoc.app/about-getting-started)
- [x] All changes are documented in the documentation app (folder `apps/docs`)
- [x] If a new component is added, at least one [Playwright screenshot test](https://github.com/SchwarzIT/onyx/actions/workflows/playwright-screenshots.yml) is added
- [x] A changeset is added with `npx changeset add` if your changes should be released as npm package (because they affect the library usage)

### CODE REVIEW

**Review by BoppLi — DISMISSED** [review]

File: .github/pull_request_template.md [hunk 1]
```diff
@@ -0,0 +1,12 @@
...
+closes #
+
+## Description
+
+<!-- Briefly describe the changes of this PR. -->
+
+## Checklist
+
+- [ ] The added / edited code has been documented with [JSDoc](https://jsdoc.app/about-getting-started)
+- [ ] All changes are documented in the documentation app (folder `apps/docs`)
```
[the remark this answers is not in the document]
Comment (larsrickert) [reply .github/pull_request_template.md:12]: We already have the changeset bot so I don't think we need this, what do you think?:
https://github.com/SchwarzIT/onyx/pull/77#issuecomment-1881003230

[the remark this answers is not in the document]
Comment (BoppLi) [reply .github/pull_request_template.md:12]: is it possible to adjust the text from the changeset bot? our "rule" that makes it concrete that `Changes on the library usage` require changesets is not clear from that text in my opinion.

[the remark this answers is not in the document]
Comment (larsrickert) [reply .github/pull_request_template.md:12]: I don't think so, I added it to the PR template

[the remark this answers is not in the document]
Comment (BoppLi) [reply .github/pull_request_template.md:12]: Nice text! thank you! 😊

**Review by BoppLi — APPROVED** [review]

**Pull request merged**